vendors/pyproject.toml

```diff
@@ -22,6 +22,7 @@ classifiers = [
 python = "^3.8"
 
 fastjsonschema = "^2.18.0"
+importlib-resources = { version = ">=1.3", python = "<3.9" }
```

Comment:
`<3.9` is not helpful here: importlib-resources has to be either vendored with poetry-core, or not.  

This is a decision that is made at `make vendor/update` time: it is not desirable that maintainers and contributors using different python versions get different answers